apps/website/src/components/Anchor.tsx

```diff
@@ -0,0 +1,13 @@
+import { FiLink } from '@react-icons/all-files/fi/FiLink';
+
+export function Anchor({ href }: { href: string }) {
+	return (
+		<a
+			aria-label="Anchor"
```

Comment:
I don't think that is necessary, native HTML elements already have proper ARIA tags applied.